crawl-ref/source/ability.cc

```diff
@@ -1420,21 +1420,24 @@ static bool _check_ability_possible(const ability_def& abil, bool quiet = false)
         return false;
     }
 
-    vector<text_pattern> &actions = Options.confirm_action;
-    if (!actions.empty())
+    if (!quiet) 
     {
```

Comment:
Or you can manually fix the trailing whitespace - it's just a single space after this `if`.